R/sapflow-Rs.Rmd

```diff
@@ -218,24 +218,59 @@ ggplot(wx_dat, aes(DOY, PAR)) +
   geom_point(aes(y = Daytime_PAR, color = PAR_group))
 ```
 
+```{r sapwood-area-calc, echo = FALSE}
+
+inventory %>% 
+  filter(Plot == "Control", Species_code != "FAGR") %>% 
+  drop_na(Sapflux) %>% 
+  select(Sapflux, DBH_2019, Species_code) %>% 
+  rename(Tree = Sapflux, DBH = DBH_2019, Species = Species_code) -> js_trees
+
+# To calculate sapflux, sapwood area needs to be calculated from sapwood depth
+sapwood_depth <- read_csv("~/Desktop/sapwood_depthsSERC.csv", 
```

Comment:
This will break on my system–put file in repository and refer to it there.

Reply:
This is the important one!

Reply:
meant to change that - ty for catching this!